aviator_object_tagging.yml

```diff
@@ -0,0 +1,16 @@
+spruce:
+  - base: ci/utility/meta.yml
+    prune:
+      - meta
+    merge:
+      - with_in: ci/utility/
+        regexp: ".*yml"
+      - with_in: ci/utility/jobs/
+        regexp: ".*yml"
+    to: aviator_pipeline_object_tagging.yml
+fly:
+  name: pdm-object-tagging
+  target: utility
+  expose: true
+  check_creds: true
+  config: aviator_pipeline_object_tagging.yml
```

Comment:
just for future reference, there is no need to change this, the file is a temporary one, so can be shared between different pipelines